doc/index.rst

```diff
@@ -37,6 +37,7 @@ QGreenland Documentation
 
     tutorials/get-started.md
     tutorials/analyze-ice-sheet-volume.md
+    tutorials/create-print-layout.md
```

Comment:
Does this work equally well with `:glob:` enabled and `tutorials/*`? We'd lose control of order, I suppose, and that's important here.

Reply:
Right. I'd like to keep `get-started` at the top. Just tested this, and it seems to work!
```
tutorials/get-started.md
tutorials/*
```